dockerfiles/run/spring/destination/conf/sqlResources.xml

```diff
@@ -258,458 +326,466 @@
             [recip] [varchar] (255) NOT NULL,
             [count] [int] NOT NULL,
             [create_time] [datetime] NOT NULL,
             PRIMARY KEY (ipaddress,sender,recip)
-            )
-        </sql>
-        <sql name="createGreyListTable" db="oracle">
+        )
+    </sql>
+    <sql name="createGreyListTable" db="oracle">
             CREATE TABLE greylist (
             ipaddress varchar2(20) NOT NULL,
             sender varchar2(255) NOT NULL,
             recip varchar2(255) NOT NULL,
             count int NOT NULL,
             create_time datetime NOT NULL,
             PRIMARY KEY (ipaddress,sender,recip)
-            )
-        </sql>
-        <sql name="createGreyListTable" db="postgresql">
+        )
+    </sql>
+    <sql name="createGreyListTable" db="postgresql">
             CREATE TABLE greylist (
             ipaddress varchar (20) NOT NULL,
             sender varchar (255) NOT NULL,
             recip varchar (255) NOT NULL,
             count int NOT NULL,
             create_time timestamp NOT NULL,
             PRIMARY KEY (ipaddress,sender,recip)
-            )
-        </sql>
-        <sql name="createGreyListTable" db="sapdb">
+        )
+    </sql>
+    <sql name="createGreyListTable" db="sapdb">
             CREATE TABLE greylist (
             ipaddress varchar (20) NOT NULL,
             sender varchar (255) NOT NULL,
             recip varchar (255) NOT NULL,
             count int NOT NULL,
             create_time date NOT NULL,
             PRIMARY KEY (ipaddress,sender,recip)
-            )
-        </sql>
-        <sql name="createGreyListTable" db="db2">
+        )
+    </sql>
+    <sql name="createGreyListTable" db="db2">
             CREATE TABLE greylist (
             ipaddress varchar (20) NOT NULL,
             sender varchar (255) NOT NULL,
             recip varchar (255) NOT NULL,
             count int NOT NULL,
             create_time timestamp NOT NULL,
             PRIMARY KEY (ipaddress,sender,recip)
-            )
-        </sql>
-        <sql name="createGreyListTable" db="ingres">
+        )
+    </sql>
+    <sql name="createGreyListTable" db="ingres">
             CREATE TABLE greylist (
             ipaddress varchar (20) NOT NULL,
             sender varchar (255) NOT NULL,
             recip varchar (255) NOT NULL,
             count int NOT NULL,
             create_time date NOT NULL,
             PRIMARY KEY (ipaddress,sender,recip)
-            )
-        </sql>
-        <sql name="createGreyListTable" db="derby">
+        )
+    </sql>
+    <sql name="createGreyListTable" db="derby">
             CREATE TABLE greylist (
             ipaddress varchar (20) NOT NULL,
             sender varchar (255) NOT NULL,
             recip varchar (255) NOT NULL,
             count int NOT NULL,
             create_time timestamp NOT NULL,
             PRIMARY KEY (ipaddress,sender,recip)
-            )
-        </sql>
-    </sqlDefs>
+        )
+    </sql>
+</sqlDefs>
```

Comment:
I don't think the indent of the block was wrong, as it's still wrapped by the `sqlResources` tag. Can you check the indent in this file again please?

Reply:
The reason for this is that i took `server/app/src/main/resources/sqlResources.xml` as a "master" file, since it seemed like it would make sense - that's why I called this an "alignment" of the SQL resources. Consequently, both files' indentation should be fixed, right?

Reply:
I get your point... Then I believe indentation was probably wrong on those other files from the start, but that's not a big problem... could be done in an other quick PR in worse case. I'm ok with that for now

Reply:
Added the indentation to the JAMES-2828 branch, as it has already got the OID change ([commit](https://github.com/apache/james-project/pull/173/commits/a69a00077ea2cd0fb294b7de0be798ce625e4882)).